srcpkgs/python3-pywebview/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'python3-pywebview'
+pkgname=python3-pywebview
+version=3.5
+revision=1
+wrksrc="pywebview-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3
+$(vopt_if gtk 'python3-cairo python3-gobject webkit2gtk')
+$(vopt_if qt 'python3-PyQt5-webengine qt5-webkit')"
```

Comment:
`qt5-webkit` should be `python3-PyQt5-webkit`, but only one of `webengine` and `webkit`  would be needed at a time.